src/core/AutomationPattern.cpp

```diff
@@ -158,18 +167,22 @@ void AutomationPattern::setTension( QString _new_tension )
 
 const AutomatableModel * AutomationPattern::firstObject() const
 {
-	AutomatableModel * m;
-	if( !m_objects.isEmpty() && ( m = m_objects.first() ) != NULL )
+	QMutexLocker m(&m_patternMutex);
+
+	AutomatableModel * model;
```

Comment:
Since you fixed some other formatting, would you mind attaching this * to the left or right? 😅